xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -636,9 +836,15 @@
         <ReturnType>System.Buffers.ReadOnlySequence&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the value of the last processed token as a ReadOnlySpan&lt;byte&gt; slice of the input payload. If the JSON is provided within a ReadOnlySequence&lt;byte&gt;
+            and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span.
+            Otherwise, <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains the token value.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+            If <see cref="P:System.Text.Json.Utf8JsonReader.HasValueSequence" /> is <see langword="false" />, <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains useless data, likely for
+            a previous multi-segment token. Therefore, only access <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> if <see cref="P:System.Text.Json.Utf8JsonReader.HasValueSequence" /> is true.
```

Comment:
My comment was meant for this line instead.

ValueSpan here is supposed to be ValueSequence, correct @ahsonkhan?

I'm also changing to here to talk about the empty value instead of useless data.

Amd what about ValueSpan, is it also empty by default instead of having useless data?

Reply:
Yes, it should be ValueSequence (not ValueSpan).

> I'm also changing to here to talk about the empty value instead of useless data.

That 's better.

> Amd what about ValueSpan, is it also empty by default instead of having useless data?

 Yes.

If HasValueSequence is true, then ValueSequence contains the data and ValueSpan is empty. Otherwise, ValueSpan contains the data and ValueSequence is empty.